### DISCUSSION TITLE
blog/common/why-vite-is-so-fast

### DISCUSSION THREAD

**Discussion by giscus** [jacob-lcs/FE-BLOG#41] [Announcements]
# blog/common/why-vite-is-so-fast

Vite 可以说是次世代的打包工具，底层基于 esbuild，打包速度大幅度提升，极大地提升了开发幸福感。Vite 是怎么做到的呢？

https://www.lcs.show/blog/common/why-vite-is-so-fast#comments-container

**Comment by jacob-lcs** [comment]
该文章也在掘金平台发布了，如果帮到了你欢迎点个关注～❤️
[文章链接](https://juejin.cn/post/7110057521519263758)

**Comment by Psycho-Monster** [comment]
讲的很好👍
